### PR TITLE
GH-480 WiFi Signal Strength

### PR DESCRIPTION
**Pull request by jamesmontemagno**
### Description of Change ###

Finalize WiFi signal strength API

### Bugs Fixed ###

- Related to issue #480 

Provide links to issues here. Ensure that a GitHub issue was created for your feature or bug fix before sending PR.

### API Changes ###

List all API changes here (or just put None), example:

Added: 
 
- SignalStrength - SignalStrength Connectivity.WiFiSignalStrength { get; } 

### Behavioral Changes ###

Describe any non-bug related behavioral changes that may change how users app behaves when upgrading to this version of the codebase.

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Updated documentation ([see walkthrough](https://github.com/xamarin/Essentials/wiki/Documenting-your-code-with-mdoc))

### CODE REVIEW

**Comment by jamesmontemagno** [comment]
build

File: DeviceTests/DeviceTests.Shared/Connectivity_Tests.cs [hunk 1]
```diff
@@ -13,5 +13,15 @@ public class Connectivity_Tests
         [Fact]
         public void ConnectionProfiles() =>
             Assert.True(Connectivity.ConnectionProfiles.Count() > 0);
+
+        [Fact]
+        public void WiFiSignale()
```
Comment (mattleibow) [inline DeviceTests/DeviceTests.Shared/Connectivity_Tests.cs]:
```suggestion
        public void WiFiSignal()
```

**Comment by jamesmontemagno** [comment]
So, it looks like iOS is going to cause a big issue here. There is no API in there that returns the active wifi signal. The HotspotHelpers needs special requirements from Apple and you only get back information about your WiFi connection itself if you are managing it. The only option here is to probably put "Unknown" for iOS unless we try to call into a Unix property of some sort.

**Comment by jamesmontemagno** [comment]
After further investigation and discussion on this topic we are not going to add this to Xamarin.Essentials at this time due to no iOS API being available. Since this is a core platform and developers expect each platform supported to work with all APIs out of the box.

**Pull request closed without merging**